questionnaires/evening_questionnaire/evening_questionnaire_armt.json

```diff
@@ -0,0 +1,704 @@
+            {
+                "code": "5",
+                "label": "5 "
+            },
+            {
+                "code": "6",
+                "label": "6 "
+            },
+            {
+                "code": "7",
+                "label": "7 "
+            },
+            {
+                "code": "8",
+                "label": "8 "
+            },
+            {
+                "code": "9",
+                "label": "9 "
+            },
+            {
+                "code": "10",
+                "label": "10 very bored"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "y",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "precipitan",
+        "form_name": "evening_questionnaire",
+        "section_header": "Seizure precipitants",
+        "field_type": "checkbox",
+        "field_label": "Have you been experiencing any special feeling in the last 24 hours?",
+        "select_choices_or_calculations": [
+            {
+                "code": "1",
+                "label": "headache "
+            },
+            {
+                "code": "2",
+                "label": "concentration difficulty "
+            },
+            {
+                "code": "3",
+                "label": "difficulty with thoughts "
+            },
+            {
+                "code": "4",
+                "label": "any other feeling"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "y",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "other",
+        "form_name": "evening_questionnaire",
+        "section_header": "",
+        "field_type": "notes",
+        "field_label": "If any other feeling, please specify",
+        "select_choices_or_calculations": "",
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "y",
+        "custom_alignment": "RH",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "seizure",
```

Comment:
Seeing as `field_name` needs to be unique, i would suggest changing the field name for all the questions in this questionnaire to something more specific like `evening_questionnaire_seizure`. Currently, the field_names are very generic.